resources/views/contests/voting-multi.blade.php

```diff
@@ -0,0 +1,60 @@
+{{--
+    Copyright 2016 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General P;ublic License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+@extends('contests.base')
+
+@section('contest-content')
+    <div class="contest__description">{!! Markdown::convertToHtml($contest->description_voting) !!}</div>
+    @if ($contest->voting_ends_at !== null && $contest->voting_ends_at->isPast())
+        <div class='contest__voting-notice'>{{trans('contest.voting.over')}}</div>
+    @endif
+    <div class='contest__accordian' id='contests-accordian'>
+        @foreach ($contests as $c)
+            <div class='panel contest__group{{ $loop->first ? ' panel-default' : '' }}'>
+                <a href="#{{$c->id}}" class='contest__group-heading' data-toggle='collapse' data-parent='#contests-accordian' aria-expanded='{{ $loop->first ? 'true' : 'false' }}'>
+                    <span>{!! $c->name !!}</span>
+                    <i class="contest__section-toggle fa fa-fw fa-chevron-down"></i>
+                </a>
+                <div class='contest__multi-panel collapse{{ $loop-> first ? ' in' : '' }}' id="{{$c->id}}">
+                    @if ($c->type == 'art')
+                        <div class="js-react--contestArtList" data-src="contest-{{$contest->id}}"></div>
+                    @else
+                        <div class="js-react--contestList" data-src="contest-{{$contest->id}}"></div>
+                    @endif
+                    <script id="contest-{{$contest->id}}" type="application/json">
+                        {!! $c->defaultJson(Auth::user()) !!}
+                    </script>
+                    @if ($c->type == 'beatmap' && $c->extra_options['beatmapset_dl'])
```

Comment:
If `extra_options` is null it doesn't error, but if it's `{}` it does...
i.e. `null['wut'] => null`, not error. TIL ![](http://static.myconan.net/psyduck.gif)